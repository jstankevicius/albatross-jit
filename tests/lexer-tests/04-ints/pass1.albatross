return 10 + 10 + 1 + 99 + 473 + 9823497 + 1;
