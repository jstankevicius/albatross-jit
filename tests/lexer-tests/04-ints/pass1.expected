1 1 RETURN
8 1 INT 10
11 1 PLUS
13 1 INT 10
16 1 PLUS
18 1 INT 1
20 1 PLUS
22 1 INT 99
25 1 PLUS
27 1 INT 473
31 1 PLUS
33 1 INT 9823497
41 1 PLUS
43 1 INT 1
44 1 SEMICOLON
